src/edit/dompos.js

```diff
@@ -104,6 +104,7 @@ function findByOffset(node, offset, after) {
   }
 }
 
+// :: (node: DOMNode, offset: number) → {node: DOMNode, offset: number}
```

Comment:
Use a single colon, not `::`, on non-public APIs, since the double colon will cause them to show up in the reference docs.